mantidimaging/gui/windows/recon/presenter.py

```diff
@@ -304,6 +306,7 @@ def _on_volume_recon_done(self, task):
             self.view.show_error_dialog(f"Encountered error while trying to reconstruct: {str(task.error)}")
             return
 
+        self._replace_inf_nan(task.result)
```

Comment:
Can you add a comment here that this is needed because pyqtgraph has problems with arrays containing inf